guides/alerting/alerting.md

```diff
@@ -33,19 +38,64 @@ SMTP_USER=user
 SMTP_PASSWORD=password
 ```
 
+The other way of configuring alerting is via the Web interface or
+through the API. To be able to use this variant, _all_ of the
+aforementioned env-vars should be left undefined when starting
+Trento. Then, configuring the alerting values could be done in a
+self-explanatory manner from Settings menu in Trento web
+console. There is also an OpenAPI spec for the API.
+
+Currently **SMTP** is the **only supported delivery mechanism** for
+alert notifications.
+
 ## Enabling Alerting at a later stage
-If your current Trento installation has Alerting disabled, you can enable it by upgrading the helm deployment.
+
+If no env-vars are in place, then you can freely change alerting
+configuration via WebUI and/or RESTful API at any time without needing
+to restart.
+
+If your current Trento installation has alerting env-vars in-place,
```

Comment:
What about: 
 If the Trento deployment uses alerting environment variables, any changes require a restart of the service.